server/src/MetadataFetch.hs

```diff
@@ -45,18 +45,20 @@ registryFromJsonFile path = do
 continuousFetch :: TVar Service.MetadataServiceRegistry -> IO ThreadId
 continuousFetch var = do
   manager <- newTlsManager
-  threadId <- forkIO $ forever $ sleepThenUpdate manager fidoAllianceRootCertificate var
+  registry <- fetchRegistry manager
+  atomically $ modifyTVar var (<> registry)
+  threadId <- forkIO $ forever $ sleepThenUpdate manager var
```

Comment:
Could we not also change the `sleepThenUpdate` function to `updateThenSleep`?

Reply:
I did that initially, but then I thought it would be better for the first update to happen synchronously (as in, before the function exits), so that the server can only start after it has some metadata